frontend/public/components/graphs/prometheus-graph.tsx

```diff
@@ -42,16 +42,18 @@ export const PrometheusGraphLink_: React.FC<PrometheusGraphLinkProps> = ({
 
   const params = new URLSearchParams();
   params.set('query0', query);
-
+  const prometheusTenancyBaseURLPresent = !!window.SERVER_FLAGS.prometheusTenancyBaseURL;
```

Comment:
Per @spadgett's https://github.com/openshift/console/pull/5661#discussion_r435263070, this would always be true. So I don't think we should be making any change in this file.